flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/TableConfig.java

```diff
@@ -32,9 +32,9 @@
 public class TableConfig {
 
 	/**
-	 * Defines the timezone for date/time/timestamp conversions.
+	 * Defines the zone id for timestamp with local time zone.
 	 */
-	private TimeZone timeZone = TimeZone.getTimeZone("UTC");
+	private ZoneId zoneId = ZoneId.systemDefault();
```

Comment:
Will this break the old behavior?

Reply:
This never be used in release-1.8.

Reply:
Another concern is it may caused unstable tests if `systemDefault` is based on some system level settings.

Reply:
In theory, tests should pass always, if the test not set zone id, the assert should not expect some time zone dependents.

Reply:
Is there any bad experience with time zone set to "UTC" by default?

Reply:
TimestampWithLocalTimeZone will be same to TimestampWithoutZone if we set to "UTC" by default.
I take a look to hive, spark, they all are default system time zone by default. Oracle maybe same.
I think user want to default system time zone when use TimestampWithLocalTimeZone at the most of the time.

Reply:
I'll change to getLocalTimeZone